main.go

```diff
@@ -205,6 +207,19 @@ func setupReconcilers(ctx context.Context, mgr ctrl.Manager) error {
 		return fmt.Errorf("setting up GCPCluster controller: %w", err)
 	}
 
+	if feature.Gates.Enabled(capifeature.MachinePool) {
+		setupLog.Info("Enabling MachinePool reconcilers")
+		gcpMachinePoolConcurrency := gcpMachineConcurrency // FUTURE: Use our own flag while feature-gated?
```

Comment:
+1 to use a seperate flag :-)

Reply:
I suggest we don't do this yet (while the controller is experimental)